ecmascript/ast/src/typescript.rs

```diff
@@ -741,7 +741,18 @@ pub enum TsLit {
     BigInt(BigInt),
 
     #[tag("TemplateLiteral")]
-    Tpl(Tpl),
+    Tpl(TsTplLitType),
```

Comment:
Why did you change this to `TsTuplLitType`?

Reply:
`Tpl` is for expression-based template literal, not for type-based, eg: `Tpl` should appear as RValue.

Reply:
I got it, thanks for the explanation.